newIDE/app/src/ProjectManager/index.js

```diff
@@ -783,6 +787,14 @@ export default class ProjectManager extends React.Component<Props, State> {
                   this.props.onOpenResources();
                 }}
               />,
+              <ListItem
+                key="extension-properties"
+                primaryText={<Trans>Extensions Properties</Trans>}
+                leftIcon={<Tune />}
+                onClick={() =>
+                  this.setState({ extensionsPropertiesDialogOpen: true })
+                }
+              />,
```

Comment:
I'd like to avoid having a new menu item for this. For the end-user, we don't really care that this is an "extension" property. If I want to change a global setting like the project name or the ad mob id, it should be at a single place... the existing project properties dialog :)